otaclient/app/configs.py

```diff
@@ -94,7 +94,7 @@ class BaseConfig:
     DOWNLOAD_RETRY: int = 10
     DOWNLOAD_BACKOFF_MAX: int = 3  # seconds
     MAX_CONCURRENT_TASKS: int = 128
-    MAX_DOWNLOAD_THREAD: int = 8
+    MAX_DOWNLOAD_THREAD = 3
```

Comment:
Emmm basically I found that 8 threads is too much and gain not so much improvement over only use 3 download threads. That is not related to bug fix though, if this change needed to be introduced separately, I can revert it can introduce it later.

Reply:
I think sometimes these configurations were changed in the previous PR w/o reason.
It is OK if the value is appropriate, but we are not sure about it  is appropriate or not if there is no reason/comment.